superset-frontend/src/dashboard/components/Header/HeaderActionsDropdown/index.jsx

```diff
@@ -88,6 +88,9 @@ const MENU_KEYS = {
 
 const DropdownButton = styled.div`
   margin-left: ${({ theme }) => theme.gridUnit * 2.5}px;
+  span {
+    color: ${({ theme }) => theme.colors.grayscale.base};
```

Comment:
I think this opens up the discussion whether we should prefer passing the color down using the Icons prop `iconColor` or do as you are doing here.

In this case, though, this is the default color of the icons and I think this should be set as the default in the Icons component instead of having to repeat it all the time. What do you think?

Reply:
We pretty much decided to let it inherit the color from the parent since most case present that way.